modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/GridDhtTxLocal.java

```diff
@@ -244,13 +244,6 @@ public void nearFutureId(IgniteUuid nearFutId) {
         return cacheCtx.isDht() && isNearEnabled(cacheCtx) && !cctx.localNodeId().equals(nearNodeId());
```

Comment:
What about adding generics for variables of `GridCacheSharedContext`, `GridCacheContext`, `GridDhtTxFinishFuture`, `IgniteInternalFuture` classes to remove warnings? At least we can add `<?>` and `<?, ?>` without any problems here.

Reply:
> current goal is just to apply IDEA's tips
One of IDEA's tips is to add generics. There is nothing to refactor, just add <?> to variables declarations in places, where IDEA complains.

Reply:
Correct, but the goal is to move step by step. 
Initial goal is to finalize fields and remove everything unused.
This should help to keep patches as simple as posslible.

The main review question - is the patch ok, not - is it complete. Can we aggree on that?